parent-checkstyle/pom.xml

```diff
@@ -39,6 +39,10 @@ limitations under the License.
           <!-- declared in parent -->
         </dependencies>
         <configuration>
+          <sourceDirectories>
+            <sourceDirectory>${project.build.sourceDirectory}</sourceDirectory>
+<!--            <sourceDirectory>${project.build.testSourceDirectory}</sourceDirectory>-->
```

Comment:
Is this to pave the way for us to run CheckStyle on the tests should we wish to do so?

Reply:
Yes

Reply:
Great.